### ISSUE TITLE
Intrinsic Charm

### ISSUE THREAD

**Issue by AleCandido** [N3PDF/eko#57]
Seems like the presence of intrinsic charm will affect also the evolution.

So in `apfel/Evolution/initPDFs.f`:
```fortran
238          if(IntrinsicCharm.and.Nf_FF.lt.4)then
  1             do alpha=0,nin(igrid)
  2                do ifl=5,6
  3                   f0ph(ifl,alpha)  = 0d0
  4                   f0ph(-ifl,alpha) = 0d0
  5                enddo
  6                f0lep(3,alpha)  = 0d0
  7                f0lep(-3,alpha) = 0d0
  8             enddo
```

**Comment by AleCandido** [comment]
We discovered how intrinsic charm is affected by evolution:
- light keep evolving with lights
- intrinsic (heavy) PDF do not evolve, since they have not collinear singularities, so the EKO in this sector is a simple identity (rank 4 identity, i.e. $\matbb{1}_fl \otimes \matbb{1}_interp$)
- for VFNS: do the same simply area by area

To do:
- [ ] name properly a function to go from Evolution Basis Light -> Reduced Flavor Basis and the opposite (will be the main engine powering the VFNS borders)
- [ ] document them properly, with pictures like: 
![image](https://user-images.githubusercontent.com/10830876/97020044-5d7dec80-1551-11eb-8bb6-d4ac9ef1efc0.png)
properly drawn...

**Comment by felixhekhorn** [comment]
closed by #61
